dashboard/lib/services/lti.rb

```diff
@@ -68,7 +68,7 @@ def self.get_claim_from_list(id_token, keys_array)
   end
 
   def self.get_claim(id_token, key)
-    id_token[key] || id_token.dig(Policies::Lti::LTI_CUSTOM_CLAIMS.to_sym, key)
+    id_token[key] || id_token.dig(Policies::Lti::LTI_CUSTOM_CLAIMS.to_sym, key) || id_token.dig(Policies::Lti::LTI_RESOURCE_LINK_CLAIM.to_sym, key) || id_token.dig(Policies::Lti::LTI_NRPS_CLAIM.to_sym, key)
```

Comment:
I'm not sure we should modify get_claim like this. I think it's meant to get a property at the top level or in custom_claims, but I'm thinking we shouldn't start adding more props to dig into. Sorry for misleading here - I think we should keep get_claim as it was, and only use it if it's useful on lines 124 & 125 in the controller